### PR TITLE
:wrench: Drain queues when empty

### PR DESCRIPTION
**Pull request by markysoft**
If zero items are placed on the queue, the drain function does not fire.
Theses changes will call the 'drain' function if no items are queued to ensure normal processing continues.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+0.2.1 / TBA
+==================
+- Call drain for an empty queue
+- Use NHSUK bunyan logger
+
```
Comment (markysoft) [inline CHANGELOG.md]: If this can be merged today the changes can become part of 0.2.0

Comment (st3v3nhunt) [reply CHANGELOG.md]: It's ok being part of `0.2.0` whenever it gets merged. The date of the `0.2.0` release needs to be updated to Tuesday i.e. `2017-10-31` as it hasn't been released.

Comment (markysoft) [reply CHANGELOG.md]: Great, I'll combine the versions and change the date.

File: package.json [hunk 2]
```diff
@@ -63,6 +63,7 @@
     "azure-storage": "^2.4.0",
     "bunyan": "^1.8.12",
```
Comment (st3v3nhunt) [inline package.json]: `bunyan` can be removed from the explicit dependencies.

**Pull request merged**